_layouts/project-page.html

```diff
@@ -13,6 +13,17 @@
   <link rel="stylesheet" type="text/css" href="{{ '/css/print.css' | prepend: site.baseurl }}" media="print">
 
   <!-- <link rel="" href="{{ '/favicon.ico' | prepend: site.baseurl }}" type="image/x-icon"> -->
+  <script type="text/javascript">
```

Comment:
I suggest putting this in project-page-functions.js as well, as we are trying to move away from putting stuff in the template file as much as possible so that when we push everything instantly works (doesn't have to be re-templated). You can put it in its on function that is called from the onProjectPageLoaded() function, or at the top of the file.